keyboards/inett_studio/sqx/hotswap/config.h

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Please add a license header to this file.

You can find a template in [`quantum/template/base/keyboard.c`](https://github.com/qmk/qmk_firmware/blob/088888937a805c29304753a1f211845744d249e8/quantum/template/base/keyboard.c#L1-L15).